com.unity.ml-agents/Runtime/Inference/BarracudaModelParamLoader.cs

```diff
@@ -282,13 +308,18 @@ static IEnumerable<string> CheckOutputTensorPresence(Model model, int memory)
             for (var sensorIndex = 0; sensorIndex < sensorComponents.Length; sensorIndex++)
             {
                 var sensorComponent = sensorComponents[sensorIndex];
-                if (!sensorComponent.IsVisual())
+                if (sensorComponent.IsVisual())
```

Comment:
Should we stop using IsVisual and start referring to the dimension instead? Feels weird to have the mix here.